bindings/cs/rl.net/LiveModel.cs

```diff
@@ -71,6 +77,24 @@ public bool TryChooseRank(string actionId, string contextJson, RankingResponse r
             return result == NativeMethods.SuccessStatus;
         }
 
+        public bool TryChooseRank(string actionId, string contextJson, ActionFlags flags, out RankingResponse response, ApiStatus apiStatus = null)
+        {
+            response = new RankingResponse();
+            return this.TryChooseRank(actionId, contextJson, flags, response, apiStatus);
+        }
+
+        public bool TryChooseRank(string actionId, string contextJson, ActionFlags flags, RankingResponse response, ApiStatus apiStatus = null)
+        {
+            int result = LiveModelChooseRankEx(this.NativeHandle, actionId, contextJson, (uint)flags, response.NativeHandle, apiStatus.ToNativeHandleOrNullptr());
```

Comment:
> apiStatus.ToNativeHandleOrNullptr() [](start = 124, length = 36)

Since apiStatus defaults to null we should not throw NullPtrExceptions. Please change in all methods to have null-conditional operators.
```csharp
apiStatus?.ToNativeHandleOrNullptr()
```

Reply:
It is not throwing right now. Please see ToNativeHandleOrNullptr here: https://github.com/VowpalWabbit/reinforcement_learning/blob/master/bindings/cs/rl.net/Native/Global.cs